_topic_maps/_topic_map_rosa_hcp.yml

```diff
@@ -149,8 +141,6 @@ Topics:
 #   Dir: cloud-experts-getting-started
```

Comment:
@EricPonvelle I think you'll need to add this back in so the other `cloud-experts-getting-started-*` modules will work